src/components/MermaidDash.js

```diff
@@ -392,6 +394,30 @@ class MermaidDash extends Component {
     return new Set(result).size;
   }
 
+  calDistance(item1, item2) {
+    return turfDistance(point(item1), point(item2), { units: 'meters' });
+  }
+
+  getUniqueSiteCount(array) {
+    let duplicateCount = 0;
+    const totalSitesCount = array.length;
+    const coordinatesArr = array.map(({ geometry: { coordinates } }) => {
+      return coordinates;
+    });
+
+    for (let i = 0; i < totalSitesCount; i++) {
+      if (i !== totalSitesCount - 1) {
+        for (let y = i + 1; y < totalSitesCount; y++) {
+          if (this.calDistance(coordinatesArr[i], coordinatesArr[y]) < 150) {
+            duplicateCount += 1;
+          }
+        }
+      }
+    }
+
+    return totalSitesCount - duplicateCount;
+  }
+
   getTransectCount(array, key) {
     const protocols = array.map(item => {
       return item.properties[key];
```

Comment:
I know this is not part of this PR, but did you know that is the same as:

```js
const protocols = array.map(item => item.properties[key])
```

🌮  - Don't have to make a change here. Just a fun fact!